### PR TITLE
Move the gfx examples here

### PR DESCRIPTION
**Pull request by csherratt**

### CODE REVIEW

**Comment by bjz** [comment]
Can we have some script to build them all at once?

**Comment by csherratt** [comment]
Sure.

**Comment by bjz** [comment]
Also travis

**Comment by csherratt** [comment]
Can you enable Travis support for this repo? I will add the .travis.yaml script.

**Comment by bjz** [comment]
done

File: build_all [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+(cd cube && cargo build)
+(cd deferred && cargo build)
+(cd glutin && cargo build)
+(cd performance && cargo build)
+(cd terrain && cargo build)
+(cd triangle && cargo build)
```
Comment (bjz) [inline build_all]: Note that ravis is taking its time building this... :(

**Comment by bjz** [comment]
I will try to make it work with cargo

**Pull request merged**